src/libraries/System.Text.Json/gen/Resources/Strings.resx

```diff
@@ -141,4 +141,16 @@
   <data name="MultipleJsonConstructorAttributeTitle" xml:space="preserve">
     <value>Type has multiple constructors annotated with JsonConstructorAttribute.</value>
   </data>
+  <data name="MultipleJsonExtensionDataAttributeFormat" xml:space="preserve">
+    <value>Type '{0}' has multiple members annotated with 'JsonExtensionDataAttribute'.</value>
```

Comment:
It's not consistent whether you quote time names or not. Two messages here have quotes and two do not. I'd say in general we don't quote unless possibly if it's an insertion.
```suggestion
    <value>Type '{0}' has multiple members annotated with JsonExtensionDataAttribute.</value>
```

Reply:
There's some others quoted in this file as well. Do what you will :)

Reply:
I thought that we also quote names if we don't want translation to convert them, e.g. type names.

Reply:
In these cases there is no way anyone would attempt to translate them as they're obviously not words. Maybe if it was `Date` but then there is a way to annotate the resource so they don't localize a part...